apps/aens/src/aens_claim_tx.erl

```diff
@@ -195,6 +195,7 @@ for_client(#ns_claim_tx{nonce     = Nonce,
                         ttl       = TTL,
                         account   = Account}) ->
     #{<<"vsn">>       => version(),
+      <<"data_schema">> => <<"NameClaimTxObject">>, % swagger schema name
```

Comment:
All of those are great catches. This corresponds to [swagger def](https://github.com/aeternity/epoch/blob/master/config/swagger.yaml#L3833)